GOVERNANCE.md

```diff
@@ -0,0 +1,149 @@
+
+Anyone can be a Contributor. You need no formal approval from the
+project, beyond the legal forms.
+
+### How to Become a Contributor
+
+* Review the coding standards to ensure your contribution is in line
+  with the project's coding and styling guidelines.
+
+* Sign the Individual CLA, or have your organization sign the Corporate CLA.
+
+* Submit your code as a PR with the appropriate DCO sign-off.
+
+## Committers
+
+Project Committers have merge access on the Rez GitHub repository
+and are responsible for approving submissions by Contributors.
+
+### Committer Responsibilities
+
+Typical activities of a Committer include:
+
+* Helping users and novice contributors.
+
+* Ensuring a response to questions posted to the ASWF rez slack channel.
+
+* Contributing code and documentation changes that improve the project.
+
+* Reviewing and commenting on issues and pull requests.
+
+* Ensuring that changes and new code meet acceptable standards and are
+  in the long-term interest of the project.
+
+* Participation in working groups.
+
+* Merging pull requests.
+
+### How to Become a Committer
+
+Any existing Committer can nominate an individual making significant
+and valuable contributions to the Rez project to become a new
+Committer.  New committers are approved by vote of the TSC.
+
+If you are interested in becoming a Committer, contact the TSC at ???
+
+TODO: best way to contact the TSC? Publicly? Privately?
+
+## Technical Steering Committee
+
+The Technical Steering Committee (TSC) oversees the overall technical
+direction of Rez, as defined in the project
+[charter](ASWF/charter/Rez-Technical-Charter.md).  This
+charter defines the TSC member terms and succession policies.
+
+TODO: Is there a rez charter? I don't know whether it belongs here or in a
+charter, but there should be some sort of mission statement that includes
+goals of increasing adoption of rez and growing the ecosystem of tools and the community of developers around rez.
+
+The responsibilities of the TSC include:
+
+* Coordinating technical direction of the project.
+
+* Project governance and contribution policy.
+
+* GitHub repository administration.
+
+* Maintaining the list of additional Committers.
+
+* Appointing representatives to work with other open source or open
+  standards communities.
+
+* Discussions, seeking consensus, and where necessary, voting on
+  technical matters relating to the code base that affect multiple
+  projects.
+
+* Coordinating any marketing, events, or communications regarding the
+  project.
+
+The TSC elects a Chair person, who acts as the project manager,
+organizing meetings and providing oversight to project
+administration. The Chair is elected by the TSC.  The Chair also
+serves as the Rez representative on the Academy Software
+Foundation (ASWF) Technical Advisory Council (TAC). The chair
+represents the project at ASWF TAC meetings.
+
+### Current TSC Members
+
+* Allan Johns (chair, author) - Nvidia
```

Comment:
I think we should either list all employers or none?

Reply:
done